### PR TITLE
Add agent whitelist variables and thresh max memory

### PR DESCRIPTION
**Pull request by mhoppal**

### CODE REVIEW

File: monasca/README.md [hunk 1]
```diff
@@ -493,7 +495,7 @@ Parameter | Description | Default
 `thresh.wait.retries` | Number of startup connection attempts to make before giving up | `24`
 `thresh.wait.delay` | Seconds to wait between retries | `5`
 `thresh.wait.timeout` | Attempt connection timeout in seconds | `10`
-`thresh.heap_ratio` | JVM heap ratio of cgroup limits | `.65`
+`thresh.memory_ratio` | JVM jvm ratio of cgroup limits | `.85`
```
Comment (timothyb89) [inline monasca/README.md]: > JVM jvm

Might be best to reword this anyway, maybe "Ratio of memory to reserve for the JVM"

File: monasca/templates/agent-daemonset.yaml [hunk 2]
```diff
@@ -121,4 +121,12 @@ spec:
               value: {{ .Values.agent.insecure | quote }}
             - name: HOSTNAME_FROM_KUBERNETES
               value: "true"
+            {{- if .Values.agent.cadvisor.enable_minimum_whitelist }}
```
Comment (aadams19) [inline monasca/templates/agent-daemonset.yaml]: Do we need these changes in agent-deployment too?

Comment (mhoppal) [reply monasca/templates/agent-daemonset.yaml]: nope they dont run these plugins

File: monasca/README.md (hunk 1 above)
Comment (mhoppal) [reply monasca/README.md]: oops!

**Pull request merged**